container-search/src/main/java/com/yahoo/search/Query.java

```diff
@@ -1093,7 +1093,7 @@ public int encodeAsProperties(ByteBuffer buffer, boolean encodeQueryData) {
         return Collections.<String,Boolean>emptyMap();
     }
 
-    private int computeTraceLevelForBackend() {
+    public int getTraceLevelForBackend() {
```

Comment:
I don't think this is a good method to have in the Query API, how about copying it to ProtobufSerialization?

Reply:
Fully agree. However it is used 2 places. Is there a no publicapi place we can put utilities like this. If I copy it you can be sure it will diverge.

Reply:
But the other place will go away.

Reply:
Moved, both the reference to dispatch.rpc and the exiting one to fs4 will go away soonish.

Reply:
Make the method private once the fs4 usage is removed, or just duplicate the logic now. Make sure to remove the previous changes to abi-check file.